all-in-one/jaeger-all-in-one-template.yml

```diff
@@ -23,25 +23,23 @@ items:
 - apiVersion: extensions/v1beta1
   kind: Deployment
   metadata:
-    name: jaeger-all-in-one
+    name: jaeger-deployment
```

Comment:
I think that this template/service/deployment/whatever should say that deployed unit is all-in-one/standalone distribution.